heron/downloaders/src/java/com/twitter/heron/downloader/Registry.java

```diff
@@ -26,6 +26,7 @@
   static {
     DOWNLOADERS.put("http", HttpDownloader.class);
     DOWNLOADERS.put("https", HttpDownloader.class);
+    DOWNLOADERS.put("distributedlog", Downloader.class);
```

Comment:
we don't support 'dlog://' in distributedlog now. if we want to support that, it requires a change on distributedlog itself.